Gemfile.lock

```diff
@@ -1,123 +1,61 @@
 GEM
   remote: https://rubygems.org/
   specs:
```

Comment:
I didn't see it specifically mentioned in the OP, so to be clear: did you remove `activesupport`, `mini_portile`, etc. because they're dependent on `nokogiri`?

Reply:
thanks @zacharysarah  for the thorough checking!!!

So I explicitly removed `jemoji`, which *implicitly* removes any of its dependencies that aren't used anywhere else --

(stuff on left is dependent on stuff on right)

```
`jemoji` ___ `gemoji`
              \___ `html-pipeline` ___ `nokogiri` ___ `mini_portile2`
             \                |
              \___ `activesupport`___ `minitest`
```

`minima` is actually still in the `Gemfile`, it's a bit further down.